src/library.cpp

```diff
@@ -2477,6 +2517,35 @@ int lammps_set_variable(void *handle, char *name, char *str)
   return err;
 }
 
+/* ---------------------------------------------------------------------- */
+
+/** Set the value of an internal-style variable.
+ *
+ * This function assigns a new value value to an internal-style variable.
```

Comment:
Please add a line of context.  E;g. This is the way to set a variable in LAMMPS which stores a numeric value by passing it a numeric value.